src/screens/EPub/controllers/file-builders/EPubFileBuilder.js

```diff
@@ -173,7 +174,7 @@ class EPubFileBuilder {
     let imgStart = text.indexOf('<img');
     let imgEnd = text.indexOf('</div>');
     let sub1 = text.substring(0, imgStart-4);
-    let sub2 = "<a href='http://localhost:3000/video?id=".concat(sourceId,"&begin=",curTime,"'>\n").replace('&', '&amp;');
+    let sub2 = "<a href='https://classtranscribe.illinois.edu".concat(links.watch(sourceId),"&begin=",curTime,"'>\n").replace('&', '&amp;');
```

Comment:
Don't hard code it, instead you should be able to extract the host from the current window URL